dspy/adapters/types/image.py

```diff
@@ -27,6 +28,17 @@ class Image(pydantic.BaseModel):
         "extra": "forbid",
     }
 
+    def format(self) -> Union[list[dict[str, Any]], str]:
+        try:
+            url = self.url
+            if isinstance(url, str) and "<DSPY_IMAGE_START>" in url and "<DSPY_IMAGE_END>" in url:
```

Comment:
nit: should we make `<DSPY_IMAGE_START>` and `<DSPY_IMAGE_END>` a const?

Reply:
This is a condition check for legacy code's compatibility, which we can remove in 1-2 version, so i would prefer not exposing a file-level constant.